<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnHistograma">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Histograma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPastel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>180</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Grafica de pastel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBarras">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>180</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Diagrama-barras</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPoligono">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Poligono-frecuencias</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOjiva">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>180</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Ojiva</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
